.wordlist.txt

```diff
@@ -10,20 +10,26 @@ EBS
 EKS
 favicon
 Furo
+Git
+GitHub
 Grafana
+hostname
```

Comment:
Let's keep that one out - is it "hostname" or "host name"? For LXD, I insist on "host name".

Reply:
Yes, that's exactly how I do it as well.
I don't mind other projects allowing "hostname" in body text as well, but then it needs to go into the custom word list. ;)

Reply:
Thanks for reviewing it! I removed it from the list :)